src/common/tests/btree_direct.c

```diff
@@ -1038,6 +1042,10 @@ sk_btr_perf(void **state)
 	}
 	now = dts_time_now();
 	D_PRINT("delete = %10.2f/sec\n", key_nr / (now - then));
+	for (i = 0; i < key_nr; i++) {
```

Comment:
This works fine but I generally prefer to have a sibling function to the allocation function.   So you could add something like sk_btr_destroy_keys(kv, key_nr).  Plus that will remove the duplication of the for loop

Reply:
okay, good idea. I'll send the patch